lib/pod-style.js

```diff
@@ -0,0 +1,18 @@
+var Filter = require('broccoli-filter');
```

Comment:
lets use persistent-filter, as it uses persistentOutput and also offers a cache shared between restarts.